README.md

```diff
@@ -0,0 +1,226 @@
+# FiberPTS: Factory Production Tracking System
+
+## Project Summary
+
+**FiberPTS** aims to provide visibility on manufacturing operations by tracking production throughput, providing key data insights to improve labor and production efficiencies. The system utilizes Supabase, a scalable and secure backend, to manage data across various IoT devices in real-time.
```

Comment:
I believe a good addition to this README would be to add the link to our video somewhere in the beginning of the document, maybe at the project summary.